From bd29f32f6946420d446a4250b171cd2da3f7d6ca Mon Sep 17 00:00:00 2001
From: Dennis Lan <dennis.yxun@gmail.com>
Date: Mon, 20 Aug 2012 18:25:29 +0800
Subject: [PATCH] iconv: remove hardcoded linking to libaray

sys-libs/glibc-2.15-r2 already including iconv functionality
and there is not libiconv.{so,a} anymore

Signed-off-by: Dennis Lan <dennis.yxun@gmail.com>
---
 android/objs/emulator_libui/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/android/objs/emulator_libui/Makefile.am b/android/objs/emulator_libui/Makefile.am
index a454878..596d969 100644
--- a/android/objs/emulator_libui/Makefile.am
+++ b/android/objs/emulator_libui/Makefile.am
@@ -175,7 +175,7 @@ libemulator_libui_la_SOURCES = $(emulator_libui) $(main_dir)
 if WIN32
 libemulator_libui_la_LDFLAGS = -no-undefined -shared -module  -Xlinker --enable-auto-import -Xlinker -L$(top_srcdir)/common/.libs/ -Xlinker -lcommon -Xlinker -L$(top_srcdir)/android/objs/emulator_common/.libs/ -Xlinker -lemulator_common -Xlinker -L/MinGW/GnuWin32/bin -Xlinker -llibiconv2 -Xlinker -lgdi32 -Xlinker -lws2_32 -Xlinker -lkernel32 -Xlinker -lkernel32 -Xlinker -lole32 -Xlinker -lkernel32 -Xlinker -lgdi32 -Xlinker -luuid -Xlinker -luser32 -Xlinker -lshell32 -Xlinker --export-all-symbol
 else
-libemulator_libui_la_LDFLAGS = -module -L$(top_srcdir)/common -lcommon -L$(top_srcdir)/android/objs/emulator_common/ -lemulator_common  -L/usr/local/lib -liconv
+libemulator_libui_la_LDFLAGS = -module -L$(top_srcdir)/common -lcommon -L$(top_srcdir)/android/objs/emulator_common/ -lemulator_common  -L/usr/local/lib
 endif
 
 
-- 
1.7.9.rc2

